crates/astria-sequencer-relayer/src/relayer.rs

```diff
@@ -127,12 +131,12 @@ impl Relayer {
 
         Ok(Self {
             sequencer,
-            sequencer_poll_period: Duration::from_millis(cfg.block_time),
+            sequencer_block_time_ms: Duration::from_millis(cfg.block_time),
             data_availability,
             validator,
             gossip_block_tx,
             state_tx,
-            queued_blocks: Vec::new(),
+            finalization_pipeline: FinalizationPipeline::default(),
```

Comment:
seems strange to have the default, would prefer explicitly setting the `head` here

Reply:
sure, what is the genesis block hash?

Reply:
or what else would be explicitly set it to at this point if not default? another option is wrapping the head in `Option<T>`, what do you think?